### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Waog**
- remove unused import
- update code highlighting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,10 +88,10 @@ yarn add ngx-cookie-backend
 
 Then edit `app.server.module.ts` and add `CookieBackendModule.forRoot()` to imports:
 
-```
+```typescript
 /* app.server.module.ts */
 
-import { CookieService, CookieBackendService } from 'ngx-cookie';
+import { CookieBackendService } from 'ngx-cookie';
```
Comment (Waog) [inline README.md:94]:
```suggestion
import { CookieBackendModule } from 'ngx-cookie';
```

**Comment by salemdar** [comment]
@Waog Looks good to me. Can you also edit it in the `projects/ngx-cookie/README.md`? It is basically same with the one in the root. But I didn't have time to find an ideal way to share these readme files.

**Comment by Waog** [comment]
> Can you also edit it in the projects/ngx-cookie/README.md?

@salemdar seems like you already did this :smile:  :+1:

**Comment by salemdar** [comment]
@Waog yeah, but thanks anyway 👍

**Pull request merged**